packages/manager/apps/web/client/app/email-domain/upgrade/upgrade.html

```diff
@@ -0,0 +1,55 @@
+<div class="page-header mb-5">
+    <h1 data-ng-bind="$ctrl.serviceName"></h1>
+</div>
+
+<div data-ovh-alert="{{ $ctrl.alerts.main }}"></div>
+
+<div class="container-fluid px-5">
+    <oui-back-button data-sref="^">
+        <span data-translate="email_domain_upgrade_title"></span>
+    </oui-back-button>
+
+    <oui-stepper>
+        <oui-step-form data-header="{{ ::'email_domain_upgrade_step_header_offer' | translate }}"
+            data-editable="!$ctrl.loading.upgrading" data-valid="$ctrl.model.offer"
+            data-prevent-next="!$ctrl.model.offer">
+            <oui-field data-label="{{ ::'email_domain_upgrade_step_header_offer_description' | translate }}"
+                data-size="m">
+                <oui-select required name="selectOffer" id="selectOffer"
+                    data-placeholder="{{ ::'email_domain_upgrade_step_header_offer_select_placeholder' | translate }}"
+                    data-model="$ctrl.model.offer" data-items="$ctrl.offers" data-match="displayName">
+                    <span data-ng-bind="$item.displayName"></span>
+                </oui-select>
+            </oui-field>
+        </oui-step-form>
+
+        <oui-step-form data-header="{{ ::'email_domain_upgrade_step_header_duration' | translate }}"
+            data-description="{{ ::'email_domain_upgrade_step_header_duration_description' | translate }}"
+            data-on-focus="$ctrl.getDurations()" data-loading="$ctrl.loading.durations"
+            data-editable="!$ctrl.loading.upgrading" data-valid="$ctrl.model.duration"
+            data-prevent-next="!$ctrl.model.duration">
+            <oui-radio-group data-required data-model="$ctrl.model.duration">
+                <oui-radio data-ng-repeat="duration in ($ctrl.durations) track by $index" data-value="duration">
```

Comment:
Isn't there a more accurate tracker ? 
```suggestion
                <oui-radio data-ng-repeat="duration in ($ctrl.durations) track by duration" data-value="duration">
```
maybe ?